added meta html values (#3213)
plone/volto: docs/source/recipes/folder-structure.md

---
html_meta:

>   "description": "Volto is based on React, Redux, and React-Router. All of the

Reviewer: ```suggestion
  "description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

> code is located in the `src` folder."

Reviewer: ```suggestion
```

>   "property=og:description": "Volto is based on React, Redux, and React-Router. All of the

Reviewer: ```suggestion
  "property=og:description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

> code is located in the `src` folder."

Reviewer: ```suggestion
```

---
html_meta:

>   "description": ""Volto is based on React, Redux, and React-Router and follows its convention of resource location."

Reviewer: ```suggestion
  "description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```

>   "property=og:description": ""Volto is based on React, Redux, and React-Router and follows its convention of resource location."

Reviewer: ```suggestion
  "property=og:description": "Volto is based on React, Redux, and React-Router and follows its convention of resource location."
```
